src/Services/newsServices.js

```diff
@@ -23,6 +28,16 @@ const createOrUpdateNews = async (news) => {
   }
 };
 
+export const deleteNewByid = async (newsRemove) => {
+  const data = await getNews();
+  const newToRemove = data.find(news => news.id === newsRemove.categoryId)
+  try {
+    await axios.delete(`http://ongapi.alkemy.org/public/api/news/${newToRemove.categoryId}`)
+  }catch(error){
+    console.log(error)
+  }
+}
+
 export const createNewsObject = (id, name, image, createdAt) => ({id, name, image, createdAt});
 
 export { createOrUpdateNews };
```

Comment:
use this export to export the functions, add all the methods:
```
{ createOrUpdateNews, deleteNewsById, getNews, getNewById }
```

Reply:
The create and update actions should have a individual function

Reply:
Perfect Sara, already fixed. Thx